packages/lwt_log/lwt_log.1.1.1/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+
+synopsis: "Lwt logging library (deprecated)"
+
+version: "1.1.1"
+license: "LGPL"
+homepage: "https://github.com/ocsigen/lwt_log"
+doc: "https://github.com/ocsigen/lwt_log/blob/master/src/core/lwt_log_core.mli"
+bug-reports: "https://github.com/ocsigen/lwt_log/issues"
+
+authors: [
+  "Shawn Wagner"
+  "Jérémie Dimino"
+]
+maintainer: "Anton Bachin <antonbachin@yahoo.com>"
+dev-repo: "git+https://github.com/ocsigen/lwt_log.git"
+
+depends: [
+  "dune" {build}
```

Comment:
```suggestion
  "dune" {>= "1.0"}
```
Dune should not be tagged as build dependency anymore (see #14266)

Reply:
Thanks. Why should there be a constraint on Dune? It doesn't have any release before 1.0.0.

Reply:
it doesn't have to be. You can choose to not have any constraints if you prefer. It's just a way to signal you don't let it without any constraints because you don't know what is its lowest-bound constraint